plugin-gradle/src/main/java/com/diffplug/gradle/spotless/SpotlessPlugin.java

```diff
@@ -35,8 +35,6 @@ public void apply(Project project) {
 		if (project.hasProperty(SPOTLESS_MODERN)) {
 			project.getLogger().warn("'spotlessModern' has no effect as of Spotless 5.0, recommend removing it.");
 		}
-		// make sure there's a `clean` and a `check`
-		project.getPlugins().apply(BasePlugin.class);
```

Comment:
From the [gradle docs](https://docs.gradle.org/current/dsl/org.gradle.api.plugins.PluginManager.html#org.gradle.api.plugins.PluginManager:apply(java.lang.Class)):

> Applies the given plugin. Does nothing if the plugin has already been applied.